src/table/virtualized-table/types/index.ts

```diff
@@ -58,7 +58,7 @@ export interface HeaderProps {
   pageInfo: PageInfo;
   forwardRef: React.RefObject<VariableSizeList<any>>;
   columns: Column[];
-  columnWidth: number[];
+  columnWidths: number[];
```

Comment:
Should say that I have removed columnWidths as a property from these component in my branch. They are now coming from the useSelectorContext instead. Which I think makes more sense

Reply:
ok sure, that works for me